ircmaxell/random-lib/lib/RandomLib/Generator.php

```diff
@@ -32,6 +32,66 @@
 class Generator {
 
     /**
+     * @const Flag for uppercase letters
```

Comment:
Mhm… They have the same value upstream as well. Let me fix this here and submit a pull request upstream as well.